src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -375,3 +375,26 @@ The following example illustrates the contexts part of the <Link to="/sdk/event-
   }
 }
 ```
+
+## Current Culture Context
+
+Current Culture Context describes certain properties of the culture in which 
+the software is used.
+
+The `type` and default key is `"current culture"`.
+
+`calendar`
+
+: _Optional_. For example `GregorianCalendar`. Free form string. 
+
+`displayName`
+
+: _Optional_. Human readable name of the current culture. For example `English (United States)`
+
+`locale`
+
+: _Optional_. Two letter code of the current culture. For example `en-US`
```

Comment:
In .NET it's:

> The name is a combination of an ISO 639 two-letter lowercase culture code associated with a language and an ISO 3166 two-letter uppercase subculture code associated with a country or region.
```suggestion

So maybe we change to:
: _Optional_. The name identifier, usually following the RFC 4646. For example `en-US` or `pt-BR`.
```

Reply:
already exists under `device.language`, we'd have to move it

Reply:
@ueman what's about deprecating `device.language` and moving to culture? I guess it'd make sense

Reply:
See 
- https://github.com/getsentry/develop/pull/329#issuecomment-836457964
- https://github.com/getsentry/develop/pull/329#issuecomment-836485242